### DISCUSSION TITLE
Did a accepted & merged PR for a project but I'm not shown as a contributor ?

### DISCUSSION THREAD

**Discussion by TheMadSword** [community/community#48185] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

Hi,

So I made a commit that was merged in a main branch of a project (lichess-org/lila ; https://github.com/lichess-org/lila/pull/12401 ) and I'm wondering why am I not considered by Github as a contributor for that project (visible in https://github.com/lichess-org/lila/graphs/contributors ; however I am visible in … "Pulse" ? ) ?

I did read https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile , and :

- over 24h passed (since merge of my commit was merged in master)
- master is the default
- I used the Github noreply email given (also visible when adding .patch)
- The commit (with my Github alias) is visible from the commit list, and when "blame"-ing, you can see my alias

Any idea why I do not appear in the contributors ?

Thanks !

p.s. : 
It seems really similar to https://github.com/orgs/community/discussions/23379 ; and following what the doc says ( "Commits must be made with an email address that is connected to your account on GitHub.com, **or the GitHub-provided noreply email address provided to you in your email settings**, in order to appear on your contributions graph.", I'd believe that using the noreply email doesn't prevent the contribution from being counted toward the project and showing the commiter as contributor. 

This seems to not be explicitly said ( for instance not mentioned in @ https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address ); unless I am missing something ? Should this be considered a bug ?

**Comment by CodeMaster7000** [comment, marked as answer]
I would suggest double-checking whether you have been included or not. What I think is most likely the problem is that you have not committed much to the repository. GitHub shows only the top 100 contributors for a project - if there are more than 100, the other contributors beyond these top 100 are never shown. Even if you set a date range so that there are less than 100 contributors displayed for that period, the list still only contains contributors who are among the all-time top 100. So, what is most likely the case is that your changes consisted of fewer additions/deletions compared to the top 100 who have contributed the most to the repo.

**Reply by TheMadSword** [reply]
Oh wow; didn't know that (@ filter not really filtering, after top 100). It's a weird limitation but understandable.

Thank you :) !
